doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/data/save_transformation.html

```diff
@@ -8,7 +8,7 @@
 	<h1>Save the transformation result</h1>
 	<p>
 		If you have defined a mapping and loaded source data in the
-		application, <i>hale�studio</i> performs the transformation derived from the
+		application, <i>hale»studio</i> performs the transformation derived from the
```

Comment:
There still seems to be some kind of encoding change as the line is marked as changed. I cannot assess if this could cause issues in how it is displayed, or not.